cocos/platform/android/jni/JniImp.cpp

```diff
@@ -104,13 +104,16 @@ namespace
     }
 }
 
+void cocos_jni_env_init (JNIEnv* env);
+
 Application* cocos_android_app_init(JNIEnv* env, int width, int height);
 
 extern "C"
 {
     JNIEXPORT jint JNI_OnLoad(JavaVM *vm, void *reserved)
     {
         JniHelper::setJavaVM(vm);
+        cocos_jni_env_init(JniHelper::getEnv());
```

Comment:
the `JNI_OnLoad` is called when load native library, and `cocos_android_app_init `was called at here when creator 1.9.x, AnySDK init JNI environment at this moment.

but on creator2.0, `cocos_android_app_init` was changed to another location, I think the reason is to get screen width and height